docs/03_server/03_request-server/04_examples.md

```diff
@@ -10,8 +10,26 @@ tags:
 ---
 
 
+Below is an example of a file where the single `requestReply` code block includes a `where` clause. You can find out more about `where` clauses on the [Basics](../../../server/request-server/basics/#where-block) page.
```

Comment:
John, I believe it is worth adding "Request Server" somewhere in these phrases, otherwise I believe that we would encounter the same problem that we have today. If I try to ask for a requestReplay example, It'll bring me one.

Reply:
Opa! Very good point. Thanks. Acting on this soon.